### PR TITLE
Movies section

### PR DESCRIPTION
**Pull request by Sarukei**
Initial progress for the movie section areas of the home page. Just the HTML & CSS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+					<div class="movie-list-wrapper">
+						<div class="movie-list">
+
+							<div class="movie">
+								<img
+									src="https://m.media-amazon.com/images/M/MV5BY2ZlNWIxODMtN2YwZi00ZjNmLWIyN2UtZTFkYmZkNDQyNTAyXkEyXkFqcGdeQXVyODkzNTgxMDg@._V1_.jpg"
+									alt="Movie cover image" class="movie__poster">
+								<div class="movie__details">
+									<h4 class="movie__title">Mortal Kombat</h4>
+									<span class="movie__rating">R</span>
```
Comment (Quinton-H) [inline index.html]: Everything honestly looks good from what I can see, my only small nitpick would be to scoot the "R" rating over a bit so it doesn't look so far away from everything else other than that excellent job.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,4400 @@
+{
```
Comment (Quinton-H) [inline package-lock.json:1]: Did you write all of this?

Comment (young-george) [reply package-lock.json:1]: package.json and package-lock.json are automatically generated by npm

Comment (Quinton-H) [reply package-lock.json:1]: The goat as appeared, I need to practice reviewing code but everything here looks good so far as I see it

**Pull request merged**